frontend/packages/ceph-storage-plugin/src/components/ocs-install/ocs-request-data.ts

```diff
@@ -42,7 +51,10 @@ export const getOCSRequestData = (
     },
     spec: {
       manageNodes: false,
-      storageDeviceSets: [createDeviceSet(scName, storage, true)],
+      encryption: {
+        enable: encrypted,
+      },
```

Comment:
```suggestion
      encrypted
```

Reply:
As per design doc, we have spec.encryption.enable for the `StorageClusterSpec`
```
type StorageClusterSpec struct {
...
	Encryption EncryptionSpec `json:"encryption,omitempty"`
...
}

type EncryptionSpec struct {
	// +optional
	Enable bool `json:"enable,omitempty"`
}
```
Then as per @leseb PR (https://github.com/rook/rook/blob/7999c32d5a508467509877a3a1dd2fede8c81862/pkg/apis/rook.io/v1/types.go) 
We have Spec.StorageClassDeviceSet.encrypted 
```
type StorageClassDeviceSet struct {
	Name                 string                     `json:"name,omitempty"`                 // A unique identifier for the set
	Count                int                        `json:"count,omitempty"`                // Number of devices in this set
	Resources            v1.ResourceRequirements    `json:"resources,omitempty"`            // Requests/limits for the devices
	Placement            Placement                  `json:"placement,omitempty"`            // Placement constraints for the devices
	Config               map[string]string          `json:"config,omitempty"`               // Provider-specific device configuration
	VolumeClaimTemplates []v1.PersistentVolumeClaim `json:"volumeClaimTemplates,omitempty"` // List of PVC templates for the underlying storage devices
	Portable             bool                       `json:"portable,omitempty"`             // OSD portability across the hosts
	TuneSlowDeviceClass  bool                       `json:"tuneDeviceClass,omitempty"`      // TuneSlowDeviceClass Tune the OSD when running on a slow Device Class
	SchedulerName        string                     `json:"schedulerName,omitempty"`        // Scheduler name for OSD pod placement
	Encrypted            bool                       `json:"encrypted,omitempty"`            // Whether to encrypt the deviceSet
}
```
I have made the changes accordingly for storagecluster and the deviceset. Let me know if its fine

Reply:
But this depends on how we do it in OCS. I think in OCS it should not be struck rather it can be key: value ?

@aruniiird need your input on this

Reply:
Ignore as discussed the operator will take care for this one.

Reply:
@cloudbehl , we just require one global boolean flag and there should not be any other dependency on the OCS operator.